presto-docs/src/main/sphinx/sql/show-roles.rst

```diff
@@ -0,0 +1,15 @@
+==========
+SHOW ROLES
+==========
+
+Synopsis
+--------
+
+.. code-block:: none
+
+    SHOW ROLES [ FROM catalog ]
+
+Description
+-----------
+
+List the roles in ``catalog`` or in the current catalog.
```

Comment:
None of the other show commands have examples, I don't think it makes sense to have them